common/src/main/java/com/microsoft/identity/common/adal/internal/net/DefaultConnectionService.java

```diff
@@ -64,7 +64,7 @@ public boolean isConnectionAvailable() {
                 .getSystemService(Context.CONNECTIVITY_SERVICE);
         NetworkInfo activeNetwork = connectivityManager.getActiveNetworkInfo();
         @SuppressWarnings("deprecation")
-        final boolean isConnectionAvailable = activeNetwork != null && activeNetwork.isConnectedOrConnecting() && !isNetworkDisabledFromOptimizations();
+        final boolean isConnectionAvailable = activeNetwork != null && activeNetwork.isConnectedOrConnecting();
```

Comment:
Just leaving a link to this issue here in case we ever go back and wonder "why didn't we just query `getActiveNetworkInfo()`?"

https://code.google.com/archive/p/android-developer-preview/issues/3164

Reply:
(Not an action item)